Add FAQ for Broken Preview Deployment Suffix (#1702)
zeit/docs: pages/docs/v2/platform/frequently-asked-questions.mdx

</Caption>

Once you have defined the `build` [script](https://docs.npmjs.com/misc/scripts), this error will disappear. Furthermore, it will not be displayed if you are using `package.json` purely to provide dependencies for your Serverless Functions located inside the `api` directory.

# Broken Preview Deployment Suffix

The Preview Deployment Suffix feature (available optionally on the "General" team settings page) allows for replacing the `now.sh` suffix of every freshly created deployment with a custom domain of your choice.

> In order for it to function properly, the custom domain you configured for the feature has to be:

Reviewer: ```suggestion
In order for this feature to function properly, the custom domain you configured must be:
```

- Active (not deleted)
- Available within the team that enabled the Preview Deployment Suffix
- Backed by an active wildcard certificate

> The easiest way to satisfy all of these constraints is to ensure the domain is also added to a project located within the same team. This project would ideally render a good-looking page on which people end up that will access the domain directly.

Reviewer: ```suggestion
The easiest way to satisfy all of these constraints is to ensure the domain is also added to a project located within the same team. This project would ideally contain a single `index.html` that would display when someone visits the bare domain.
```
